subset/dhcp/README.md

```diff
@@ -15,4 +15,11 @@ Due to this, the DHCP module requires the ipaddr module to run successfully:
 - Reconnect device and check for DHCP request. 
 #### Result Cases:
 - PASS: A DHCP request has been received by the device after the port has been disconnected and connected.
-- FAIL: No DHCP request has been received.
\ No newline at end of file
+- FAIL: No DHCP request has been received.
+
+## Integrated Testing
```

Comment:
You don't need to document the faux-2 behavior, you need to document that "A static IP address will results in FAIL"  -- or, really whatever will happen with the test results when ANY device (not just faux) is configured with a static IP address.

This will likely be a problem in the big picture because the result really should be SKIP in this case. But that can be sorted out later.